Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -0,0 +1,655 @@
+                        break;
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Unexecuted").FirstOrDefault();
+                        break;
+                    case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Pending:
+                        testStepLog.Status = statuses.Where(z => z.Name == "Deffered").FirstOrDefault();
+                        break;
+                }
+            }
+            else
+            {
+                testStepLog.Status = statuses.Where(z => z.Name == "Unexecuted").FirstOrDefault(); ;
+            }
+        }
+
+        public bool ExportActivitiesGroupToALM(ActivitiesGroup activitiesGroup, QtestTest mappedTest, string parentObjectId, ObservableList<ExternalItemFieldBase> testCaseFields, ref string result)
+        {
+            ConnectALMServer();
+            testcaseApi = new QTestApi.TestcaseApi(connObj.Configuration);
+
+            try
+            {
+                if (mappedTest == null) // Create new test case
+                {
+                    QTestApiModel.TestCaseWithCustomFieldResource testCase = new QTestApiModel.TestCaseWithCustomFieldResource(null, null, null, new List<QTestApiModel.PropertyResource>());
+                    testCase.Description = activitiesGroup.Description;
+                    testCase.Name = activitiesGroup.Name;
+                    testCase = testcaseApi.CreateTestCase((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), testCase);
+                    foreach (ActivityIdentifiers actIdent in activitiesGroup.ActivitiesIdentifiers)
+                    {
+                        QTestApiModel.TestStepResource stepResource = new QTestApiModel.TestStepResource(   null, null, 
+                                                                                                            ((Activity)actIdent.IdentifiedActivity).Description == null ? string.Empty : ((Activity)actIdent.IdentifiedActivity).Description,
+                                                                                                            ((Activity)actIdent.IdentifiedActivity).Expected == null ? string.Empty : ((Activity)actIdent.IdentifiedActivity).Expected);
+                        stepResource.PlainValueText = ((Activity)actIdent.IdentifiedActivity).ActivityName;                                                                   
+                        testcaseApi.AddTestStep((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), testCase.Id, stepResource);
+                    }
+                    activitiesGroup.ExternalID = testCase.Id.ToString();
+                    activitiesGroup.ExternalID2 = testCase.Id.ToString();
+                }
+                else //##update existing test case
+                {
+                    ////##update existing test case
+                    //test = ImportFromQC.GetQCTest(activitiesGroup.ExternalID);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)